helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/templates/liveindexing-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+{{- range $service := .Values.liveIndexing.services }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "alfresco-elasticsearch-connector.fullname" $ }}-{{ $service.name }}
+  labels:
+    {{- include "alfresco-elasticsearch-connector.labels" $ | nindent 4 }}
+spec:
+  {{- if eq $service.name "mediation"}}
+  replicas: 1
+  {{- else if not $.Values.autoscaling.enabled }}
+  replicas: {{ $service.replicaCount | default 1 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "alfresco-elasticsearch-connector.selectorLabels" $ | nindent 6 }}
+  template:
+    metadata:
+      {{- with $.Values.podAnnotations }}
+      annotations:
+        {{- toYaml $ | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "alfresco-elasticsearch-connector.selectorLabels" $ | nindent 8 }}
+    spec:
+      # only set this secret if a private docker registry variable is defined
+      imagePullSecrets:
```

Comment:
I have to replace the string value with a variable. I'm going to replace it during the first commit after the pull request review.